package.json

```
@@ -3,7 +3,7 @@
   "version": "2.5.0-rc.0",
   "io_backend_api": "https://raw.githubusercontent.com/pagopa/io-backend/v7.35.0-RELEASE/api_backend.yaml",
   "io_public_api": "https://raw.githubusercontent.com/pagopa/io-backend/v7.35.0-RELEASE/api_public.yaml",
-  "io_content_specs": "https://raw.githubusercontent.com/pagopa/io-services-metadata/1.0.6/definitions.yml",
+  "io_content_specs": "https://raw.githubusercontent.com/pagopa/io-services-metadata/00af3fb2b863047ac1514f7c19c8c00f98d66a13/definitions.yml",
```

```suggestion
  "io_content_specs": "https://raw.githubusercontent.com/pagopa/io-services-metadata/1.0.7/definitions.yml",
```
In order to lock the version with the latest `io-services-metadata` version